src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+const Difficulty = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
+const QuizGame = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState([]);
+  const [score, setScore] = useState(0);
+  const [isGameOver, setIsGameOver] = useState(false);
+  const [isGameStart,setIsGameStart]=useState(false)
+  const startQuiz = async () => {
+    setIsLoading(true);
+    setIsGameOver(false);
+    setIsGameStart(true)
+    const newQuestions = await fetchQuizQuestions(
+      TOTAL_QUESTIONS,
+      Difficulty.EASY
+    );
+    setQuestions(newQuestions);
+    setScore(0);
+    setUserAnswers([]);
+    setCurrentQuestionNumber(0);
+    setIsLoading(false);
+  };
+  const checkAnswer = (e) => {
+    if (!isGameOver && isGameStart) {
```

Comment:
`isGameOver `is false by default. So, `!isGameOver` will be true at first. Is it still possible for user to check answer when game is finished?

Reply:
i deleted isGameOver state.